src/Server/modules/DataClasses.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
 	},
 
 	USPlace: class {
-		constructor(state, county) {
+		constructor(state, county, currentNumConfirmed,currentNumDeaths) {
```

Comment:
We don't want currentNumConfirmed and currentNumDeaths here. if we do, we need to modify the function LoadUSData() inside modules/ReadCSV.js.

Instead, we want these variables to be populated inside the functions addNumConfirmed() and addNumDeaths() based on the values in the dictionary.